src/OpenLoco/Scenario.cpp

```diff
@@ -142,9 +149,25 @@ namespace OpenLoco::Scenario
     // 0x0049685C
     void initialiseDate(uint16_t year)
     {
-        registers regs;
-        regs.ax = year;
-        call(0x0049685C, regs);
+        // NB: this base value was already 1800 in Locomotion.
+        uint32_t baseYear = year - 1800;
+        uint32_t dayCount = (baseYear / 4) * (365 * 3 + 366);
+
+        // Leap year?
+        if ((baseYear & 3) != 0)
+            dayCount += (baseYear * 365) + 1;
```

Comment:
For a 1930 scenario, I'm getting dayCount = 94203 where the original computes 47483. Not sure what's wrong yet.